src/ShadowDOM/wrappers/events.js

```diff
@@ -797,7 +797,7 @@
 
       if (found && count === 1) {
         var target = getTargetToListenAt(this);
-        target.removeEventListener_(type, dispatchOriginalEvent, true);
+        target.removeEventListener_(type, dispatchOriginalEvent, capture);
```

Comment:
An alternative solution to this: If we need `useCapture` to always be `true`, we could rework the logic here so that we get the count of ALL bound listeners instead of `listeners[i].capture === capture`